packages/components/src/internal/renderers/AttachmentCard.tsx

```diff
@@ -122,18 +127,11 @@ export const AttachmentCard: FC<Props> = memo(props => {
                 )}
             </div>
 
-            <Modal bsSize="large" show={showModal} onHide={_hideModal}>
-                <Modal.Header closeButton>
-                    <Modal.Title>
-                        <a onClick={_onDownload} className="clickable" title={'Download ' + noun}>
-                            {title ?? name}
-                        </a>
-                    </Modal.Title>
-                </Modal.Header>
-                <Modal.Body>
+            {showModal && (
+                <Modal bsSize="lg" onCancel={_hideModal} title={modalTitle}>
```

Comment:
thoughts on if this modal should use "Dismiss" instead of "Cancel" since it is the single button case?

Reply:
Yes I think that makes it more consistent with the other modals so I'll make the change.